packages/start/src/client/StartClient.tsx

```diff
@@ -24,3 +24,13 @@ export function StartClient() {
     </Dummy>
   );
 }
+
+export function StartClientTanstack() {
```

Comment:
not sure we want a tanstack specific component? But it needs one less dummy component it seems

Reply:
Hmm.. can we for now just move this definition of StartClientTanstack to the entry-client in the tanstack example?

Reply:
having some trouble inlining this